docs/usage/configuration-options.md

```diff
@@ -345,6 +345,8 @@ If you prefer that Renovate more silently automerge _without_ Pull Requests at a
 - Automerge the branch commit if it's: (a) up-to-date with the base branch, and (b) passing all tests
 - As a backup, raise a PR only if either: (a) tests fail, or (b) tests remain pending for too long (default: 24 hours)
 
+To allow automerges on branches that are behind and green, then add `rebaseWhen="conflicted"`. This will not merge a conflicted branch.
```

Comment:
```suggestion
To allow automerges on branches that are behind and green, add `rebaseWhen="conflicted"`. 
Note that this will not merge a conflicted branch.
```